big_tests/tests/mam_send_message_SUITE.erl

```diff
@@ -0,0 +1,142 @@
+            Config2;
+        false ->
+            {skip, require_rdbms}
+    end.
+
+end_per_group(_Groupname, Config) ->
+    case mongoose_helper:is_rdbms_enabled(domain()) of
+        true ->
+            dynamic_modules:restore_modules(domain(), Config);
+        false ->
+            ok
+    end,
+    ok.
+
+group_to_modules(send_message) ->
+    MH = muc_light_helper:muc_host(),
+    [{mod_mam_meta, [{backend, rdbms}, {pm, []}, {muc, [{host, MH}]},
+                     {send_message, mam_send_message_example}]},
+     {mod_muc_light, []},
+     {mam_send_message_example, []}].
+
+load_custom_module() ->
+    mam_send_message_example:module_info(),
+    {Mod, Code, File} = code:get_object_code(mam_send_message_example),
+    rpc(mim(), code, load_binary, [Mod, File, Code]).
+
+%%%===================================================================
+%%% Testcase specific setup/teardown
+%%%===================================================================
+
+init_per_testcase(TestCase, Config) ->
+    escalus:init_per_testcase(TestCase, Config).
+
+end_per_testcase(TestCase, Config) ->
+    escalus:end_per_testcase(TestCase, Config).
+
+%%%===================================================================
+%%% Test Cases
+%%%===================================================================
+
+mam_pm_send_message(Config) ->
+    P = ?config(props, Config),
+    F = fun(Alice, Bob) ->
+        escalus:send(Alice, escalus_stanza:chat_to(Bob, <<"OH, HAI!">>)),
+        escalus:wait_for_stanza(Bob),
+        mam_helper:wait_for_archive_size(Alice, 1),
+        mam_helper:wait_for_archive_size(Bob, 1),
+        escalus:send(Alice, stanza_archive_request(P, <<"q1">>)),
+        Res = wait_archive_respond(Alice),
+        assert_respond_size(1, Res),
+        [Msg] = respond_messages(Res),
+        verify_has_some_hash(Msg)
+        end,
+    escalus_fresh:story(Config, [{alice, 1}, {bob, 1}], F).
+
+mam_muc_send_message(Config0) ->
+    F = fun(Config, Alice) ->
+        P = ?config(props, Config),
+        Room = muc_helper:fresh_room_name(),
+        MucHost = muc_light_helper:muc_host(),
+        muc_light_helper:create_room(Room, MucHost, alice,
+                                     [], Config, muc_light_helper:ver(1)),
+        escalus_assert:has_no_stanzas(Alice),
+        RoomAddr = <<Room/binary, "@", MucHost/binary>>,
+        escalus:send(Alice, escalus_stanza:groupchat_to(RoomAddr, <<"text">>)),
+        M = escalus:wait_for_stanza(Alice),
+        escalus:assert(is_message, M),
+        escalus_assert:has_no_stanzas(Alice),
+        mam_helper:wait_for_room_archive_size(MucHost, Room, 1),
+        escalus:send(Alice, escalus_stanza:to(stanza_archive_request(P, <<"q1">>), RoomAddr)),
+        [Msg] = respond_messages(assert_respond_size(1, wait_archive_respond(Alice))),
+        verify_has_some_hash(Msg)
+        end,
+    escalus_fresh:story_with_config(Config0, [{alice, 1}], F).
```

Comment:
Why `story_with_config`?

Reply:
Because we need the updated user list in `muc_light_helper:create_room`